src/Authentication/Authentication.Test/Microsoft.Graph.Authentication.Test.csproj

```diff
@@ -1,11 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
do we have separate unit test runs for .net4.x and .net core?

Reply:
Yes, our pipeline should run them separately.

![image](https://user-images.githubusercontent.com/7061532/94719328-e2d20080-0307-11eb-9a3a-26435295f950.png)